Lib/test/test_largefile.py

```diff
@@ -45,6 +45,13 @@ def tearDownClass(cls):
             raise cls.failureException('File was not truncated by opening '
                                        'with mode "wb"')
 
+    @bigmemtest(size=_2G, memuse=1)
```

Comment:
_2G doesn't look correct: it should be 'size = size + 1' no?

(By the way, I suggest to rename the global 'size' constant to something like FILESIZE, but it should be done in a different PR.)

Reply:
what do you suggest for `size = size + 1`? `2**32 + 1` ?

Reply:
I'm suggesting @bigmemtest(size=size + 1, memuse=1)